app/src/main/java/com/platypus/android/tablet/TeleOpPanel.java

```diff
@@ -1092,10 +1074,10 @@ protected void onProgressUpdate(Integer... result) {
 
       }
 
-      if (connected == true) {
-        ipAddressBox.setBackgroundColor(Color.GREEN);
+      if (connected) {
+          ipAddressBox.setBackgroundColor(Color.GREEN);
       }
-      if (connected == false) {
+      else if (!connected) {
```

Comment:
If this is a boolean, you can just do `else` instead of `else if`.  Or you can do a ternary like:
```
ipAddressBox.setBackgroundColor(connected ? Color.GREEN : Color.Red)
```